targets/targets.json

```diff
@@ -4376,7 +4391,8 @@
             "EMAC",
             "TRNG",
             "FLASH",
-            "MPU"
+            "MPU",
+            "WATCHDOG"
```

Comment:
This is not needed as target inherits from FAMILY_STM32 where WATCHDOG is already defined